components/tikv_util/src/security.rs

```diff
@@ -158,18 +162,29 @@ impl SecurityManager {
 
 struct Fetcher {
     cfg: Arc<SecurityConfig>,
+    last_fetch: Arc<Mutex<Instant>>,
```

Comment:
Pre https://github.com/tikv/tikv/pull/7251/files#r398349060

```suggestion
    cfg: Arc<Mutex<(SecurityConfig, Instant)>>,
```

Reply:
`cfg` is read-only, why mutex protection is needed ?

Reply:
How did you figure `grpc_threads * timeout`. It takes up to 1s after the certificate file is updated which means 0s~1s instead of `grpc_threads * timeout`.